src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -154,6 +168,20 @@ def __init__(self, name, button, panel, main_frame, main_data):
 
         panel.btn_pnl_user_settings.Bind(wx.EVT_BUTTON, self._toggle_user_settings_panel)
 
+        # Toolbar
+        self.tb = panel.toolbar
+        self.tab_data_model.tool.subscribe(self._on_tool_va)
+        self.tb.add_tool(TOOL_ACT_ZOOM_FIT, self.view_controller.fitViewToContent)
+        self.tb.add_tool(TOOL_RULER, self.tab_data_model.tool)
+        self.tb.add_tool(TOOL_LINE, self.tab_data_model.tool)
+
+    def _on_tool_va(self, selected_tool):
+        if selected_tool == TOOL_NONE:
+            self.vp.canvas.enable_drag()
+            return
+        if CAN_DRAG in self.vp.canvas.abilities:
+            self.vp.canvas.disable_drag()
```

Comment:
I don't think you need to change the drag capability. Normally, when the gadget overlay is active, it gets first the mouse events, and never pass them to the canvas. So the drag doesn't even start.